crates/re_types/tests/fuzzy.rs

```diff
@@ -306,7 +313,13 @@ fn roundtrip() {
         // NOTE: Keep those around please, very useful when debugging.
         // eprintln!("field = {field:#?}");
         // eprintln!("array = {array:#?}");
-        eprintln!("{} = {array:#?}", field.name);
+        if field.name == "fuzz1021" {
+            // TODO(jleibs): Fields that contain Float16 apparently don't supported fmt
+            // https://github.com/jorgecarleitao/arrow2/blob/main/src/array/primitive/fmt.rs#L35
+            eprintln!("{} = Can't be printed", field.name);
```

Comment:
Some context for the poor soul who sees this in their terminal:
```suggestion
            eprintln!("{} = Can't be printed (f16 not supported)", field.name);